src/solvers/applyexp.jl

```diff
@@ -0,0 +1,25 @@
+function applyexp_solver()
```

Comment:
This wrapper function should be removed.

Reply:
And the inner function renamed `applyexp_updater`.

Though maybe we should just get rid of `applyexp` at this point, I don't think there is any advantage of it over `KrylovKit.exponentiate`.

Reply:
I'd vote for removing it.

Reply:
Ok, let's remove it in this PR.